### PR TITLE
Nodejs/week1

### PR DESCRIPTION
**Pull request by zmzade**

### CODE REVIEW

File: nodejs/week1/homework/warmup.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+circle.getDiameter(); // 20
+console.log(
+  `circle: Diameter = ${circle.getDiameter()}, Circumference = ${circle.getCircumference()}, Area = ${circle.getArea()}`
+);
+console.log(
+  `circle 1: Diameter = ${circle1.getDiameter()}, Circumference = ${circle1.getCircumference()}, Area = ${circle1.getArea()}`
+);
+console.log(
+  `circle 2: Diameter = ${circle2.getDiameter()}, Circumference = ${circle2.getCircumference()}, Area = ${circle2.getArea()}`
+);
```
Comment (laila1222) [inline nodejs/week1/homework/warmup.js:32]: Well done, good job. :)

File: nodejs/week1/homework/src/backend/app.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  response.send(getRandom(getMealsWithReviews()));
+});
+app.get("/cheap-meals", async (request, response) => {
+  response.send(getMealsWithReviews().filter((x) => x.price < 100));
+});
+app.get("/large-meals", (request, response) => {
+  response.send(getMealsWithReviews().filter((x) => x.maxNumberOfGuests > 7));
+});
+///////////reservation
+app.get("/reservations", async (request, response) => {
```
Comment (laila1222) [inline nodejs/week1/homework/src/backend/app.js:40]: All routes work as they should, and return the right data! 👍

**Review by laila1222 — APPROVED** [review]
Excellent job on completing you first NodeJs homework.  💯

File: nodejs/week1/homework/src/backend/app.js (hunk 2 above)
Comment (zmzade) [reply nodejs/week1/homework/src/backend/app.js:40]: So thanks :)

**Pull request merged**